matterpy/contrib/rss.py

```diff
@@ -72,10 +72,22 @@ def init(manager, conf):
 
 
 def post_to_text(entry, format_string):
-    title = entry.title
-    url   = entry.link
-    body  = entry.description
```

Comment:
You can't just remove `url` and `body` because running instances would fail after updating (backwards compatibility is important!). All of `url`, `link`, `body` and `description` need to be usable in the format.

Reply:
Added str.replace() for backwards compatibility.